src/app/components/itinerary-details/itinerary-details.component.html

```diff
@@ -22,21 +22,30 @@ <h1 class="flex items-center gap-2 text-2xl font-bold">
         <p>
           Cost:
           {{
-            activity().cost | currency: activity().currency : 'symbol-narrow'
+            activity()?.cost | currency: activity()?.currency : 'symbol-narrow'
```

Comment:
No need for safety checks. At this point (after line 10) we already know `activity()` exists

Reply:
For some reason it still says the Object is possibly 'null' if I remove the check

Reply:
As discussed, resolved.